docs/higher-order/tca.ipynb

```diff
@@ -0,0 +1,1448 @@
+{
```

Comment:
<blockquote><em> In contrast to PCA, TCA can be used to identify trends in neuron activity within trials as well as *across</em> trials.</blockquote><p>TCA is a dimensionality reduction technique as PCA. However, it does not enforce strict decorrelation of its components, allow to force non-negative weights and provide a framework to analyze responses against the inherent trial structure shared by a lot of systems neuroscience experiments.</p><p> </p><p>In the end of the whole paragraph, add the following: </p><p> </p><p>We do not aim to cover all uses of TCA, but rather help build a concrete intuition on how TCA can be used and pro and cons of this approach. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>